interface/frontend/package.json

```diff
@@ -43,8 +43,8 @@
     "babel": "inherit"
   },
   "dependencies": {
-    "vue-resource": "^1.3.4",
-    "vue": "^2.5.6"
+    "vue": "^2.5.6",
+    "vue-resource": "^1.3.4"
```

Comment:
This project is using Axios instead of Vue Resource.

Reply:
vue-resoure as a dep was there from the beginning. I can remove it once we confirm there are no trace of its usage.